include/ranges.h

```diff
@@ -311,6 +310,14 @@ class id : public detail::coordinate<id<Dims>, Dims> {
 
 	constexpr id(const range<Dims>& range) : coordinate(detail::make_from, range) {}
 
+	constexpr id(const detail::zeros_t /* tag */) {}
+
+	constexpr id(const detail::ones_t /* tag */) {
+		for(int d = 0; d < Dims; ++d) {
+			(*this)[d] = 1;
+		}
+	}
+
```

Comment:
I like being able to explicitly say that I'm initializing to `zeros`, rather than requiring the reader to know that the default constructor does that. (In principle, I'd rather have no default constructor at all, but that often makes the type more difficult to use in some contexts)

Reply:
Ok, I'm still not sure whether we'll ever need the `ones` variant though.

Reply:
We discussed this offline, and while @psalz is still not fully convinced of the utility of the `ones` variant, we decided to keep it.